src/Magnum/GL/TransformFeedback.h

```diff
@@ -340,7 +340,7 @@ class MAGNUM_GL_EXPORT TransformFeedback: public AbstractObject {
          *      eventually @fn_gl{BindTransformFeedback} and
          *      @fn_gl_keyword{BindBuffersRange} or @fn_gl_keyword{BindBufferRange}
          */
-        TransformFeedback& attachBuffers(UnsignedInt firstIndex, std::initializer_list<std::tuple<Buffer*, GLintptr, GLsizeiptr>> buffers);
+        TransformFeedback& attachBuffers(UnsignedInt firstIndex, Containers::ArrayView<std::tuple<Buffer*, GLintptr, GLsizeiptr>> buffers);
```

Comment:
Same here and in the overload below:  

- the `std::initializer_list` overload needs to be kept in order to allow conveniently passing a `{...}` list
- the `std::tuple` should get replaced by a Triple, with a deprecated backwards compatibility overload that allocates a temporary `Array<Triple>` and calls the new function; the `#include <tuple>` / `#include <StlForwardTuple.h>` then get included only conditionally

Hope it's clear enough, please complain if not ;)

Reply:
Ok! I'm doing the `std::initializer_list` change in a first commit, and will do the `std::tuple` one afterwards